Podcast/EpisodeCard.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  EpisodeCard.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 2/22/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+import SwiftyJSON
+
+class EpisodeCard: Card {
+    
+    var episodeID: Int
+    var episodeTitle: String = ""
+    var dateCreated: Date!
+    var time: Double!
+    var descriptionText: String!
+    var smallArtworkImageURL: URL?
+    var smallArtworkImage: UIImage?
+    var nRecommended: Int //number total out of user base that has recommended episode
+    var tags: [String]
+    var seriesTitle: String!
+    var isBookmarked: Bool!
```

Comment:
all of these variables should have default values, false for bools, empty string for strings, in general avoid ! when possible